index.js

```diff
@@ -1,16 +1,25 @@
-var express = require('express');
-var app = express();
-var cors = require('cors')
-var path = require('path');
+const express = require('express');
+const app = express();
+const mongoose = require('mongoose')
+const cors = require('cors')
+const path = require('path');
 
 global.appRoot = path.resolve(__dirname);
 
-var PORT = 3000;
+const PORT = 3000;
 
 app.use(cors())
 app.use(express.json());
 app.use('/static', express.static(__dirname + '/public'));
 
+mongoose.connect('mongodb://localhost:27017/swc-bookings',

```

Comment:
metti le costanti fuori in un file di configurazione, se no diventa impossibile sistemarle in futuro